itl/command-plugins.conf

```diff
@@ -2505,6 +2505,20 @@ object CheckCommand "ldap" {
 			set_if = "$ldap_verbose$"
 			description = "Show details for command-line debugging"
 		}
+		"-4" = {
+			set_if = "$ldap_ipv4$"
+			description = "Use IPv4 connection"
+		}
+		"-6" = {
+			set_if = "$ldap_ipv6$"
+			description = "Use IPv6 connection"
+		}
+		"-A" = {
+      # Need to come after -T or -S
+			order = 99
```

Comment:
Have a look where other such comments are placed. If one line above as this one, indent it exactly at `order = 99`.

Reply:
updated - editor fail 😆